### PR TITLE
literature: display number_of_pages in detail view

### PR DESCRIPTION
**Pull request by Dinika**
User-story: [INSPIR-1046](https://its.cern.ch/jira/browse/INSPIR-1046)

Screenshot:
screenshot from 2018-08-06 11-10-21

Signed-off-by: Dinika Saxena <dinika.saxena@cern.ch>

### CODE REVIEW

File: src/literature/components/NumberOfPages.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class NumberOfPages extends Component {
+  render() {
+    const { numberOfPages } = this.props;
+
+    if (!numberOfPages) {
+      return null;
+    }
+
+    return <span>{numberOfPages} pages</span>;
```
Comment (drjova) [inline src/literature/components/NumberOfPages.jsx]: if there is only one page?

Comment (harunurhan) [reply src/literature/components/NumberOfPages.jsx]: I think we need to decide what way to go and do it like that everywhere, page(s) or you want to have conditional suffix everywhere?

File: src/literature/components/NumberOfPages.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class NumberOfPages extends Component {
+  render() {
+    const { numberOfPages } = this.props;
+
+    if (!numberOfPages) {
```
Comment (harunurhan) [inline src/literature/components/NumberOfPages.jsx]: this won't render, also when this value is `0`, is it what we want? @michamos

File: src/literature/components/NumberOfPages.jsx (hunk 1 above)
Comment (drjova) [reply src/literature/components/NumberOfPages.jsx]: conditional suffix everywhere to be consistent, cc @StellaCh

Comment (StellaCh) [reply src/literature/components/NumberOfPages.jsx]: I agree

File: src/literature/components/NumberOfPages.jsx (hunk 2 above)
Comment (michamos) [reply src/literature/components/NumberOfPages.jsx]: it's strictly positive according to the schema https://github.com/inspirehep/inspire-schemas/blob/5d129dcdb5dda1584381c9c253cc7d1b8a499e0f/inspire_schemas/records/hep.yml#L1173-L1176, so it should be displayed if `number_of_pages` is present.

**Comment by harunurhan** [comment]
@Dinika please rebase

**Pull request merged**